helm-charts/charts/kube-starrocks/values.yaml

```diff
@@ -66,6 +66,7 @@ starrocks:
   initPassword:
     enabled: false
     password: ""
+    # The secret name that contains password, the key of the secret is "password", and you should create it first.
```

Comment:
should move this into a separate PR.

Reply:
Ok, I will create a new PR for testing for v1.8.0